azurelinuxagent/common/cgroupapi.py

```diff
@@ -514,12 +514,16 @@ def create_cgroup(controller):
                 # There was an issue with systemd-run. We need to log it and retry the extension without systemd.
                 err_msg = 'Systemd process exited with code %s and output %s' % (e.exit_code, process_output) \
                     if isinstance(e, ExtensionOperationError) else "Systemd timed-out, output: %s" % process_output
+                event_msg = 'Failed to run systemd-run for unit {0}.scope. ' \
```

Comment:
Somehow, the way I read your sample error message, it gives me the impression that the error is coming from the retry.

Maybe 

> Failed to run systemd-run for unit {0}.scope. Will retry invoking the extension without systemd. Systemd error: {1}

Reply:
That sounds better, updating.